pom.xml

```diff
@@ -22,7 +22,7 @@
     <properties>
         <java.version>1.8</java.version>
         <biodata.version>0.4.9</biodata.version>
-        <datastore.version>0.3.3</datastore.version>
+        <datastore.version>0.3.4</datastore.version>
```

Comment:
Why not include the `mongo4` here instead of hardcoding that in different places?

Reply:
Updated!